pulsar-broker/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java

```diff
@@ -369,21 +369,16 @@ public void runStreamStorage(CompositeConfiguration conf) throws Exception {
                  .build())
             .buildAdmin()) {
 
+            LOG.info("Creating default namespace");
             try {
-                NamespaceProperties ns = FutureUtils.result(admin.getNamespace("default"));
```

Comment:
Could you explain in detail how these codes causes this exception? `io.grpc.StatusRuntimeException: NOT_FOUND`

Reply:
It was throw by 
```
private void processGetNamespaceResponse(String namespace,
                                             GetNamespaceResponse response,
                                             CompletableFuture<NamespaceProperties> getFuture) {
        StatusCode code = response.getCode();
        if (StatusCode.SUCCESS == code) {
            getFuture.complete(response.getNsProps());
            return;
        }
        getFuture.completeExceptionally(createRootRangeException(namespace, code));
    }
```